narwhal/primary/src/metrics.rs

```diff
@@ -395,6 +397,13 @@ impl PrimaryMetrics {
                 registry
             )
             .unwrap(),
+            round_propagation_latency: register_histogram_vec_with_registry!(
```

Comment:
Maybe calling this proposal_latency is more accurate. There is another path where potential parents are received by the proposer, and self.round is advanced to the parent round. It seems we are not tracking the latency for this and probably we don't need to.